package.json

```diff
@@ -28,16 +28,16 @@
     "eslint-config-babel": "^7.0.1",
     "eslint-plugin-flowtype": "^2.20.0",
     "eslint-plugin-prettier": "^2.1.2",
-    "flow-bin": "^0.41.0",
+    "flow-bin": "^0.52.0",
     "gulp": "^3.9.0",
-    "gulp-babel": "7.0.0-alpha.18",
+    "gulp-babel": "babel/gulp-babel#d8c67b",
```

Comment:
What does this syntax do? I've never seen it before 😄

Reply:
This targets a specific commit, in this case https://github.com/babel/gulp-babel/commit/d8c67b

Reply:
This will pick a specific commit, in this case: https://github.com/babel/gulp-babel/commit/d8c67b